src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -7977,7 +7986,7 @@ private void VisitThisOrBaseReference(BoundExpression node)
             var parameter = node.ParameterSymbol;
             int slot = GetOrCreateSlot(parameter);
             var parameterType = GetDeclaredParameterResult(parameter);
-            var typeWithState = GetParameterState(parameterType, parameter.FlowAnalysisAnnotations);
+            var typeWithState = GetParameterState(parameterType, parameter.FlowAnalysisAnnotations, parameter.IsNullChecked);
```

Comment:
>parameter.IsNullChecked [](http://example.com/codeflow?start=100&length=23)

Are we testing a case where this value makes a difference (from passing `false` for instance)?

Reply:
Oop, I might not have addressed this. Looking.

Reply:
I think there's not an obvious scenario where we can observe a behavior difference so I'm not going to add specific tests for it at this time.